examples/apps/ai-collab/src/app/presence.ts

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	IPresence,
+	LatestMap,
+	type PresenceStates,
+	type PresenceStatesSchema,
+} from "@fluid-experimental/presence";
+
+export interface User {
+	photo: string;
+}
+
+const statesSchema = {
+	onlineUsers: LatestMap<{ value: User }, `id-${string}`>(),
+} satisfies PresenceStatesSchema;
+
+export type UserPresence = PresenceStates<typeof statesSchema>;
+
+export function buildUserPresence(presence: IPresence): UserPresence {
```

Comment:
A comment on what this function does would be nice

Reply:
Updated